From e138dc2315cfce1dfadc01ff70947bfa7961b433 Mon Sep 17 00:00:00 2001
From: Vishal Sagar <vishal.sagar@xilinx.com>
Date: Wed, 19 Sep 2018 10:19:26 +0530
Subject: [PATCH 0412/1851] dt-bindings: display: xlnx: Add ST352 payload
 insertion in Chroma stream

commit cafddbc4b717bc3880a9b1abe442085fbb144114 from
https://github.com/Xilinx/linux-xlnx.git

Added optional property which when present says that the IP supports
ST352 payload insertion in the chroma stream.

Signed-off-by: Vishal Sagar <vishal.sagar@xilinx.com>
Reviewed-by: Hyun Kwon <hyun.kwon@xilinx.com>
Signed-off-by: Michal Simek <michal.simek@xilinx.com>
State: pending
Signed-off-by: Yaliang Wang <Yaliang.Wang@windriver.com>
---
 Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
index 54701ed3098f..093d53321e0f 100644
--- a/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
+++ b/Documentation/devicetree/bindings/display/xlnx/xlnx,sdi-tx.txt
@@ -17,6 +17,7 @@ Required properties:
 Optional properties:
  - xlnx,vpss: vpss phandle
    This handle is required only when VPSS is connected to SDI as bridge.
+ - xlnx,tx-insert-c-str-st352: Insert ST352 payload in Chroma stream.
 
 Example:
 
-- 
2.31.1

